keps/sig-network/4410-k8s-network-interface/kep.yaml

```diff
@@ -0,0 +1,48 @@
+title: k8s-network-interface
+kep-number: 4410
+authors:
+  - "@mikezappa87"
+  - "@shaneutt"
+owning-sig: sig-network
+participating-sigs:
+  - sig-network
```

Comment:
Surely at least SIG Node should be participating (there's no way this doesn't affect kubelet, CRI)..?

I would also tag Cluster Lifecycle at least as FYI / advisory since cluster lifecycle folks will know about and have suggestions re: node readiness and cluster configuration.